numba/cuda/cudadrv/driver.py

```diff
@@ -2721,8 +2721,11 @@ def add_file(self, path, kind):
             raise LinkerError(msg)
 
     def add_cu(self, path, name):
-        raise NotImplementedError("Linking CUDA source files is not supported "
-                                  "with the ctypes binding. ")
+        raise NotImplementedError("Support for fp16 or linking CUDA source "
+                                  "files requires the use of the Nvidia CUDA "
+                                  "bindings and enabling the "
+                                  "NUMBA_CUDA_USE_NVIDIA_BINDING environment "
+                                  "variable. See documentation for me details.")
```

Comment:
Suggested changes:

- `fp16` should be `float16` -I noticed that the code in Numba is unusual in Python referring to 16-bit floating point values as `fp16` rather than `float16` which seems to be used everywhere else - for example, the NumPy dtype. This is an inconsistency I should probably have realised much sooner, so I'd like to try not introducing more instances of it.
- `Nvidia` is usually written `NVIDIA`
- Instead of saying the environment variable should be enabled, say that it should be set to `1`.
- `See documentation for me details` needs the grammar correcting, and it would be helpful to provide a link to the relevant documentation. Where is the user supposed to look?